README.md

```diff
@@ -1,18 +1,142 @@
-# Cantara
+[Cantara](docs/static/img/cantara.svg 'Cantara Logo')
```

Comment:
Is it intentional to not show the logo here? Otherwise, add an exclamation mark in the beginning, like this:
```
![Cantara](docs/static/img/cantara.svg 'Cantara Logo')
```